### DISCUSSION TITLE
Help Needed with EmailJS Integration and Error 418 when sending email

### DISCUSSION THREAD

**Discussion by feloc50** [emailjs-com/help#169] [Q&A]
Hi everyone,

I'm facing an issue while using EmailJS with a Node.js script deployed on Firebase Functions. Here’s the error I’m encountering:

`Error sending email: EmailJSResponseStatus { status: 418, text: 'This SDK version is unsupported. Please check https://www.emailjs.com/docs/sdk/installation/' }
`I’m calling the EmailJS service via an API call, and I’m using the latest version of the SDK:

"@emailjs/nodejs": "^2.2.0"
I’ve reviewed the installation guide here: [EmailJS SDK Documentation](https://www.emailjs.com/docs/sdk/installation/), but I’m still stuck.

Has anyone faced a similar issue? Could you guide me on how to resolve this?

Any help would be greatly appreciated. Thanks in advance!

**Comment by xr0master** [comment]
The current version is 5.0.2
https://www.npmjs.com/package/@emailjs/nodejs

**Comment by feloc50** [comment]
Oh thanks !
